### PR TITLE
Maintenance

### PR DESCRIPTION
**Pull request by ecoologic**
Currently, the build is failing as bundler can't find the gems.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 language: ruby
 rvm:
-  - 2.2.1
+  - 2.3.1
```
Comment (lucascaton) [inline .travis.yml]: 2.3.3? 😁

File: LICENSE.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015 Ricardo Bernardeli
+Copyright (c) 2015 NetEngine
```
Comment (lucascaton) [inline LICENSE.txt]: <del>2016</del> 2017?

File: rulerr.gemspec [hunk 3]
```diff
@@ -6,8 +6,8 @@ require 'rulerr/version'
 Gem::Specification.new do |spec|
   spec.name          = "rulerr"
   spec.version       = Rulerr::VERSION
-  spec.authors       = ["Ricardo Bernardeli"]
-  spec.email         = ["ricardobcs@gmail.com"]
+  spec.authors       = ["Ricardo Bernardeli", "Erik Ecoologic"]
+  spec.email         = %w(ricardobcs@gmail.com erikecoologic@gmail.com)
```
Comment (lucascaton) [inline rulerr.gemspec:10]: The `email` attribute is optional, you can completly remove this line if you don't want to leave your email public almost asking for spam :p

**Review by lucascaton — APPROVED** [review]

File: LICENSE.txt (hunk 2 above)
Comment (ecoologic) [reply LICENSE.txt]: Should I update that too? ok

File: rulerr.gemspec (hunk 3 above)
Comment (ecoologic) [reply rulerr.gemspec:10]: yes, that's my spam email (:

**Pull request merged**